test/ipld-tests.js

```diff
@@ -150,5 +151,65 @@ module.exports = (repo) => {
         }
       ], done)
     })
+
+    describe('links are merkle paths', () => {
+      const draft = {
+        title: 'Title of the blogpost'
+      }
+
+      const alice = {
+        name: 'Alice'
+      }
+
+      const author = {
+        name: {
+          '@link': path.join('/', ipld.multihash(ipld.marshal(alice)), 'name')
```

Comment:
This will not do what we want on windows. `path.join` uses the os specific seperator, which is `\` on windows.